lib/src/signed_video_h26x_common.c

```diff
@@ -1136,7 +1136,7 @@ signed_video_create(SignedVideoCodec codec)
 
     self->last_two_bytes = LAST_TWO_BYTES_INIT_VALUE;
     for (int i = 0; i < MAX_NALUS_TO_PREPEND; i++) {
-      self->last_two_bytes_buffer[i] = LAST_TWO_BYTES_INIT_VALUE;
+      self->sei_data_buffer[i].last_two_bytes = LAST_TWO_BYTES_INIT_VALUE;
     }
```

Comment:
You can remove initializing the buffer since we do not take values from it when starting writing the SEI.